db/schema.rb

```diff
@@ -0,0 +1,64 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 2023_04_25_131813) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "foods", force: :cascade do |t|
+    t.string "name"
+    t.string "measurement_unit"
+    t.decimal "price"
+    t.integer "quantity"
+    t.bigint "user_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["user_id"], name: "index_foods_on_user_id"
+  end
+
+  create_table "recipe_foods", force: :cascade do |t|
+    t.decimal "quantity"
+    t.bigint "recipe_id", null: false
+    t.bigint "food_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["food_id"], name: "index_recipe_foods_on_food_id"
+    t.index ["recipe_id"], name: "index_recipe_foods_on_recipe_id"
+  end
+
+  create_table "recipes", force: :cascade do |t|
+    t.string "name"
+    t.integer "preparation_time"
+    t.integer "cooking_time"
+    t.text "description"
+    t.boolean "public"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.bigint "user_id", null: false
+    t.index ["user_id"], name: "index_recipes_on_user_id"
+  end
+
+  create_table "users", force: :cascade do |t|
+    t.string "name"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.string "email", default: "", null: false
+    t.string "encrypted_password", default: "", null: false
+    t.datetime "remember_created_at"
+    t.index ["email"], name: "index_users_on_email", unique: true
+  end
+
+  add_foreign_key "foods", "users"
+  add_foreign_key "recipe_foods", "foods"
+  add_foreign_key "recipe_foods", "recipes"
+  add_foreign_key "recipes", "users"
+end
```

Comment:
[Considering your team consists of 3 members](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md#:~:text=If%20your%20team%20has%203%20members%20your%20data%20model%20will%20be%20a%20little%20bit%20more%20complex%3A), you should have more than just these tables. Your schema should be identical to this ERD: 
![image](https://github.com/IsmailChangezi/recipe-rails/assets/93520178/a4b99c89-6d1b-41e8-b760-df7b0044e8a6)

[Your project should also have the following views](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md#:~:text=If%20your%20team%20has%203%20members%20these,into%20consideration%20a%20chosen%20recipe%20and%20inventory.): 
![image](https://github.com/IsmailChangezi/recipe-rails/assets/93520178/b6fe8563-4f3d-41da-9e5d-f4483374b463)

[This link](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md#:~:text=If%20your%20team%20has%203%20members%20these%20are%20requirements,food%20items%20and%20total%20price%20of%20the%20missing%20food.) will take you to the list of missing requirements that you should implement in your project.
After making these changes, you should also update your current tests and add new ones according to the changes made.

Reply:
Hi @laiifuu, One of our teammates was off and not working that's why we couldn't make it like that, what's the solution in this case?

Reply:
I'm not sure what should be done in this case. Did you talk about this with the Microverse success team? Did he have permission to take the week off? Because when I got the review request, it said 3 participants.

Reply:
No, he was in our team but just not working on his tasks